examples/Spec/Int.hs

```diff
@@ -33,7 +33,7 @@ instance HasLogicalModel IntProp Int where
   satisfiesProperty IsSmall i = i <= 10 && i >= -10
 
 instance HasParameterisedGenerator IntProp Int where
-  parameterisedGenerator s = Source $ do
+  parameterisedGenerator s = do
```

Comment:
The type of `parameterisedGenerator` has gone back to `Gen` so we no longer need the `Source` constructor here.